wlp-managed-8.5/src/main/java/org/jboss/arquillian/container/was/wlp_managed_8_5/WLPManagedContainer.java

```diff
@@ -576,10 +707,23 @@ private Element createApplication(Document doc, String deploymentName, String ar
          application.appendChild(sharedLib);
       }
 
+      if(containerConfiguration.getSecurityConfiguration() != null) {
```

Comment:
Unfortunately I cannot edit the page directly. Here is the description of the new configurations:

securityConfiguration - Within xml deployment the refered file will embedded within the application tag into the server.xml and should be used to configure the security settings.

failSafeUndeployment - In the following setup xml deployments might faile to undeploy because of a open FileHandle. OS Windows and JVM Oracle. Enabling this flag suppresses the exception to be thrown and marks the file to be delete upon exiting the JVM process.